nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ConvertJSONToSQL.java

```diff
@@ -37,6 +37,7 @@
 import java.util.Set;
 import java.util.UUID;
 import java.util.concurrent.atomic.AtomicReference;
+import org.apache.commons.lang.StringEscapeUtils;
```

Comment:
All other NiFi code that uses StringEscapeUtils refers to the Commons Lang 3 class `org.apache.commons.lang3.StringEscapeUtils`